tutorials/global_illumination.md

```diff
@@ -0,0 +1,419 @@
+
+### Voxel Cone Tracing
+
+A scene rendered with VCT is reliable and good quality, and thus is the best choice for most scenes.
+
+VCT parameters include:
+* **Resolution**
+* **Octant count**
+* **Bounce count**
+* **High quality**
+* **Anisotropic**
+* **Thin wall counter**
+* **Conserve memory**
+* **Debug visualization mode**
+
+### Cascaded Image Voxel Cone Tracing
+
+Compared to VCT, a scene rendered with CI VCT is slightly lower resolution, but the scene can be re-voxelized every frame, so it’s much faster when dealing with very large scenes. However, it is more memory-intensive than VCT.
+
+CI VCT parameters include:
+* **Bounce count**
+* **High quality**
+* **Anisotropic**
+* **Debug visualization mode**
+* **Cascade**
+    * **Resolution**
+    * **Octant count**
+    * **Thin wall counter**
+    * **Area half size**
+
+## Enabling global illumination in Gazebo
+
+During simulation, the GUI and the camera sensor view are two different scenes. The GUI scene is rendered on the frontend client process, and the camera sensor scene is rendered on the backend server process. Thus, GI can be enabled on either or both of these processes.
+
+> **Note:** GI solutions require hardware that uses DX11 or GL4. <br>
+> **Note:** CI VCT must be run with Vulkan as the render engine API backend.
+
+### How to enable global illumination for the GUI
+
+GI can be enabled for the GUI through a GUI plugin. Both VCT and CI VCT are supported for the GUI.
+
+#### Example usage for VCT
+
+1) Open the [global_illumination.sdf](
+https://github.com/gazebosim/gz-sim/blob/main/examples/worlds/global_illumination.sdf) world with
+
+```bash
+gz sim global_illumination.sdf
+```
+<div style="text-align:center;">
+    \image html files/global_illumination/gui_demo_default.png width=60%
+</div>
+<br>
+
+2) From the plugin dropdown, select the Global Illumination Vct plugin.
+
+3) On the card, check the `Enabled` button to enable GI and change the parameter values as desired.
+
+<div style="text-align:center;">
+    \image html files/global_illumination/gui_demo_with_vct.png width=60%
+</div>
+
+#### Example usage for CI VCT
+
+1) Open the [global_illumination.sdf](
+https://github.com/gazebosim/gz-sim/blob/main/examples/worlds/global_illumination.sdf) world using Vulkan with
+
+```bash
+gz sim global_illumination.sdf --render-engine-api-backend vulkan
+```
+
+2) From the plugin dropdown, select the Global Illumination Ci Vct plugin.
+
+3) On the card, check the `Enabled` button to enable GI and change the parameter values as desired.
+
+### How to enable global illumination for the sensor
+
+GI can be enabled for sensors through the `<global_illumination>` element in the Sensors System plugin. VCT is supported for the sensor.
+
+#### Example usage with VCT
+
+We will demonstrate how to enable VCT for the sensor with the SDF file below. (The finished SDF file can be viewed [here](
+https://github.com/gazebosim/gz-sim/blob/main/examples/worlds/global_illumination.sdf).)
```

Comment:
change link to point to the `gz-sim8` branch

Reply:
95530a6aa1699b232512fdd76034cc279b2a730e